### DISCUSSION TITLE
Nutrition Information Corrections

### DISCUSSION THREAD

**Discussion by bpmich** [mealie-recipes/mealie#4566] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When "Nutrition Values" is enabled, fields for various nutrition facts can be entered. These fields as displayed appear to describe nutrition information for the entire recipe. If the servings counter is updated, the ingredients update, but the Nutrition Values do not. This allows me to print a recipe with correct ingredient amounts for a given serving size, but incorrect Nutrition Values

### Please provide a concise description of the feature that would resolve your issue.

Add a separate serving size counter for the Nutrition Values section. This allows a recipe to quantify what should be cooked for the batch, and the nutrition values would quantify what is in one serving of the batch.

### Please consider and list out some caveats or tradeoffs made in your design decision

Caveats or tradeoffs would be adding an additional field which must be updated before the recipe is complete. This field may not be parsed simply from recipe providers. These caveats do not resolve the issue listed in the "description of the problem" section of this feature request.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by ChipSkylark37** [comment]
I often do not fill out the nutrition section because unless I stay strict to the portion and specific ingredient, it can be very off. Even using one "large egg" from one store vs another can be different. 

Having each ingredient (or substitution of one) have its own nutrition value that then gets "added" all together would make that section really useful, but still perfect.

It would also need to scale with adjusting the recipe overall,to .5x, 2x or any other number. What if you just changed one ingredient? Currently you have to calculate that and adjust manually.
